src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -102,6 +105,9 @@ public function testHas()
         $this->assertTrue($builder->has('bar'), '->has() returns true if a service exists');
     }
 
+    /**
+     * @group legacy
```

Comment:
You are right. I split this into several small tests with the great effect that none of them needed to be marked as legacy.